faq/github.md

```diff
@@ -197,3 +197,70 @@ If you have trouble, create an issue and tag `@@rhodyprog4ds/fall20instructors`
 ````
 Doing each new assignment in its own branch is best practice. In a typical software development flow once the codebase is stable a new branch would be created for each new feature or patch. This analogy should help you build intuition for this GitHub flow and using branches.  Also, pull requests are the best way for us to give you feedback. Also, if you create a branch when you do not need it, you can easily merge them after you are done, but it is hard to isolate things onto a branch if it's on main already.
 ````
+
+## How would I continue to add to this FAQ page by myself?
+````{toggle}
+Getting everything set up is really simple! Though it gets a little bit technical. This will describe the process to fork any remote repo and allow you to use ```git fetch``` to get information from the remote repo you created the fork from and ```git push``` to your fork (which can then be merged by the original owner you forked from).
```

Comment:
```suggestion
Getting everything set up is really simple! Though it gets a little bit technical. This will describe the process to fork any remote repo and allow you to use `git fetch` to get information from the remote repo you created the fork from and `git push` to your fork (which can then be merged by the original owner you forked from).
```

inline uses one backtick, 3 is for blocks